src/dash-table/type/formatter.ts

```diff
@@ -0,0 +1,20 @@
+import {
+    ColumnType,
+    IColumnType,
+    LocaleFormat
+} from 'dash-table/components/Table/props';
+
+import { getFormatter as getNumberFormatter } from './number';
+
+const DEFAULT_FORMATTER = (value: any) => value;
+export default (locale: LocaleFormat) =>
+    (c: IColumnType) => {
+        let formatter;
+        switch (c.type) {
+            case ColumnType.Numeric:
+                formatter = getNumberFormatter(locale, c);
+                break;
+        }
+
+        return formatter || DEFAULT_FORMATTER;
+    };
```

Comment:
formatter resolver for the columns, if no formatter is found, use one that is a value-passthrough